docs/day-2-operations/Backups.adoc

```diff
@@ -144,8 +144,105 @@ Backup the dump.rb file from redis
 oc cp $(oc get pods -l 'deploymentConfig=system-redis' -o json | jq '.items[0].metadata.name' -r):/var/lib/redis/data/dump.rdb ./system-redis-dump.rdb
 ----
 
+=== `Secrets`
+[source,bash]
+----
+oc get secrets system-smtp -o json --export > system-smtp.json
+oc get secrets system-seed -o json --export > system-seed.json
+oc get secrets system-database -o json --export > system-database.json
+oc get secrets backend-internal-api -o json --export > backend-internal-api.json
+oc get secrets system-events-hook -o json --export > system-events-hook.json
+oc get secrets system-app -o json --export > system-app.json
+oc get secrets system-recaptcha -o json --export > system-recaptcha.json
+oc get secrets system-redis -o json --export > system-redis.json
+oc get secrets zync -o json --export > zync.json
+oc get secrets system-master-apicast -o json --export > system-master-apicast.json
+----
+
+=== `ConfigMaps`
```

Comment:
I don't see the following ConfigMaps included:

```
backend-environment
mysql-extra-conf
mysql-main-conf
redis-config
system
```

Can we add them to the backup list?

Take into account that when HA databases are deployed some of them do not apply.

Reply:
All those configmaps content can be reproduced by the template call.

I am assuming we do not support the customer editing and updating those resources. Do we support it?

Reply:
As far as I know changes have to be provided via templates parameters but I don't have/I'm aware of an "official" product-level instance on this.

However, for example, the system-smtp ConfigMap has to be edited by hand at the moment as there are no parameters in the template, and I understand people might be configuring mailing in 3scale deployments. So why add that ConfigMap and not others in the backup then? It wouldn't surprise some customers have some customizations in the mysql configurations

Reply:
`system-smtp` is not in the list you provided. It is already in the backup item list. 

Those secret/configmaps can be reproduced

For instance:

`mysql-extra-conf` content is:
```
"data": {
        "mysql-charset.cnf": "[client]\ndefault-character-set = utf8\n\n[mysql]\ndefault-character-set = utf8\n\n[mysqld]\ncharacter-set-server = utf8\ncollation-server = utf8_unicode_ci\n"
    },
```

Do we support the customer updating `default-character-set` for instance?

Another example:

`mysql-main-conf`
```
"data": {
        "my.cnf": "!include /etc/my.cnf\n!includedir /etc/my-extra.d\n"
    },
```
Do we support the customer adding custom mysql config?

The content of those config maps are static and does not need to be backed-up. Unless we allow/support modification. Then it would make sense. 

wdty @miguelsorianod ?

Reply:
See my https://github.com/3scale/3scale-Operations/pull/8/files#r412418599 comment. I think we are talking about the same thing